backend/api_machine.py

```diff
@@ -16,7 +16,7 @@ def __init__(self):
         self.lifter_index = load_csv_as_list(join(self.query_root, "lifter_index.csv"))
         self.db_root = QueryThis.results_root  # meh
 
-    def lifter_totals(self, gender="male", start=0, stop=100) -> Union[list[Dict], Dict]:
+    def lifter_totals(self, gender="male", start=0, stop=99) -> Union[list[Dict], Dict]:
```

Comment:
I think it's fine here to send them all through tbh, I'll limit the number on the frontend